src/simulation/openables/GrubbyChest.ts

```diff
@@ -0,0 +1,65 @@
+import LootTable from '../../structures/LootTable';
+import SimpleOpenable from '../../structures/SimpleOpenable';
+
+const FoodTable = new LootTable({ limit: 100 })
+	.add('Egg potato', 4, 60)
+	.add('Shark', 4, 35)
+	.add(
+		[
+			['Saradomin brew(2)', 3],
+			['Super restore(2)', 1]
+		],
+		1,
+		5
+	);
+
+const PotionTable = new LootTable({ limit: 100 })
+	.add(
+		[
+			['Super attack(2)', 1],
+			['Super strength(2)', 1],
+			['Super defence(2)', 1]
+		],
+		1,
+		34
+	)
+	.add(
+		[
+			['Super defence(2)', 1],
+			['Ranging potion(2)', 1]
+		],
+		1,
+		34
+	)
+	.add('Super restore(3)', 2, 16)
+	.add('Prayer potion(3)', 2, 16);
+
+const GrubbyChestTable = new LootTable()
+	/* Food roll */
+	.every(FoodTable, 2)
+
+	/* Potion roll */
+	.every(PotionTable, 1)
+
+	/* Main roll */
```

Comment:
place a `TODO` here to check back occasionally to see if the wiki has exact rates